shopelectro/context.py

```diff
@@ -31,18 +29,15 @@ def __init__(self, page, tags: newcontext.Tags):
         self._tags = tags
 
     def context(self):
-        def template_context(page, tag_titles, tags):
-            return {
-                'page': page,
-                'tag_titles': tag_titles,
-                'tags': tags,
-            }
-
         tags_qs = self._tags.qs()
-        self._page.get_template_render_context = partial(
-            template_context, self._page, tags_qs.as_title(), tags_qs
-        )
-
+        # use dirty patch here, because it's the most simple method
```

Comment:
it seems it isn't relevant comment. let's say something here about `display` descriptor here just in case